doc/.readthedocs.yaml

```
@@ -9,11 +9,18 @@ version: 2
 build:
   os: ubuntu-22.04
   tools:
+    golang: "1.22"
     python: "3.11"
-  jobs:
-    pre_install:
-      - cd doc && python3 .sphinx/build_requirements.py
+  commands:
       - git fetch --unshallow || true
+      - cd doc && make integrate
+      - cd doc/integration/lxd/ && go build -ldflags "-s -w" -o trimpath -o lxc.bin ./lxc
+      # Pretend that woke is installed - we don't need it for building
+      # (workaround until https://github.com/canonical/microovn/pull/168 is merged
+      # and https://github.com/canonical/microceph/pull/400 is restored)
+      - ln -s /bin/sh doc/integration/microovn/docs/woke
+      - ln -s /bin/sh doc/integration/microceph/docs/woke
```

This type of kludge typically use a less feature-full binary like `true`:
```suggestion
      - ln -s /bin/true doc/integration/microovn/docs/woke
      - ln -s /bin/true doc/integration/microceph/docs/woke
```